design/select/index.jsx

```diff
@@ -62,6 +58,8 @@ const Select = ({
         inputRef.current.blur()
         break
 
+      case KEYS.RIGHT:
+      case KEYS.LEFT:
```

Comment:
I guess we misunderstood each other. I expected you to remove left and right at all. When you press left or right arrow key, text cursor should go (iterate) over text (user input). It should not be prevented.